lazyflow/utility/helpers.py

```diff
@@ -250,7 +250,45 @@ def __iter__(self):
         retSlice = [(self.toSlice(src,True),self.toSlice(trgt,True),self.toSlice(mask)) \
                     for src,trgt,mask in retRoi]
         return retSlice.__iter__()
- 
+
+
+def get_default_axisordering(shape):
+    """Given a data shape, return the default axis ordering
+
+    For data types that do not support axistags, we assume a default axis
+    ordering, given the shape, and implicitly the number of dimensions.
+
+    Args:
+        shape (tuple): Shape of the data
+
+    Returns:
+        str: String, each position represents one axis.
+    """
+    axisorders = {
+        2: 'yx',
+        3: 'zyx',
+        4: 'zyxc',
+        5: 'tzyxc'
+    }
```

Comment:
I'm not sure, but I think it's more common to see tiff files that use `cyx` order instead of interleaved channels (`yxc`).  Similarly, I think `tzcyx` or `tczyx` might be more common than ilastik's internal default of `tzyxc`.

(I wonder if we can find any example multi-channel files in the wild to corroborate or dispute this claim of mine....)